lib/SiftResponse.php

```diff
@@ -21,9 +22,14 @@ public function __construct($result, $httpStatusCode, $request) {
             $this->body = $json->decode($result);
         }
         $this->httpStatusCode = $httpStatusCode;
+        $this->curlErrno = $curlErrno;
         $this->request = $request;
         $this->rawResponse = $result;
 
+        if ($curlErrno !== 0) {
+            $this->apiErrorMessage = "curl: ($curlErrno) $curlError";
```

Comment:
This matches the command line `curl` error message, e.g. `curl: (6) Could not resolve host: badsubdomain.example.com`